ports/sdl-shadercross/portfile.cmake

```
@@ -0,0 +1,42 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO libsdl-org/SDL_shadercross
+    REF e1000de7d174af8f84935db9a59b365d1ae55d32
+    SHA512 ef0a167fdc9f4903719132f4d4108d286fe29f0773788730106cc2b41fa551d0920f9fc332e2c17b1aed2442a04d4abf84983a36c456191a5f7bb7e8c1cf9f8d
+    HEAD_REF main
+)
+
+if(NOT VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_find_acquire_program(BISON)
+    get_filename_component(BISON_DIR "${BISON}" DIRECTORY)
+    vcpkg_add_to_path(PREPEND "${BISON_DIR}")
+endif()
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DSDLSHADERCROSS_SPIRVCROSS_SHARED=OFF
+        -DSDLSHADERCROSS_INSTALL=ON
+        -DSDLSHADERCROSS_INSTALL_CMAKEDIR_ROOT=share/${PORT}
+)
+
+vcpkg_cmake_install()
+
+set(config_path "share/${PORT}/SDL3_shadercross")
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+    set(config_path "share/${PORT}")
+endif()
+vcpkg_cmake_config_fixup(CONFIG_PATH "${config_path}")
```

The following is what I meant in my previous comment.
I think this should work, as the correct install directory is set for every target platform in `vcpkg_cmake_configure` via `-DSDLSHADERCROSS_INSTALL_CMAKEDIR_ROOT=share/${PORT}`
```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME "SDL3_shadercross" CONFIG_PATH "share/${PORT}")
```